pulpcore/pulpcore/app/models/task.py

```diff
@@ -221,8 +221,10 @@ class Task(Model):
     non_fatal_errors = JSONField(default=list)
     error = JSONField(null=True)
 
-    parent = models.ForeignKey("Task", null=True, related_name="spawned_tasks")
-    worker = models.ForeignKey("Worker", null=True, related_name="tasks")
+    parent = models.ForeignKey("Task", null=True, related_name="spawned_tasks",
+                               on_delete=models.SET_NULL)  # TODO: Correct behavior?
```

Comment:
Is parent being used anywhere? If not, maybe delete this FK?

Reply:
AFAIK "parent" is used to keep track of tasks that were spawned by other tasks, e.g. sync with auto-publish.  The publish task would set the sync task as its parent, and the sync task would then show the publish task as a child when you GET the record.

Reply:
Yea, I know what the field does but AFAICT, we don't have any tasks in our code yet that use it. If that's the case, I'd probably remove this FK until we actually need it and then add it back.

Reply:
Nothing is setting it, but since other code is looking at it, better just to leave it there IMO.